reledmac.dtx

```diff
@@ -6739,14 +6739,14 @@
 % \end{macro}
 % \begin{macro}{\flag@start@RTL}
 % \begin{macro}{\flag@end@RTL}
-% With \XeLaTeX, there is a problem when using RTL: the writing of command in the numbered auxiliary files (\verb+.1+, \verb+.2+ etc) is reversed when the first argument of \cs{edtext} is typset in one line, but is NOT reversed when this first argument is typset in two lines or more.\footnote{This problem is linked to the way \XeTeX\ manages right-to-left typsetting. David Carlisle explains it on \url{http://tex.stackexchange.com/a/333373/7712} and provides a potential solution, using \cs{vadjust}. However, in real test case, this add spurious vertical space in reledmac. That is why we use the solution explained below.}
+% With \XeLaTeX, there is a problem when using RTL: the writing of a command in the numbered auxiliary files (\verb+.1+, \verb+.2+ etc) is reversed when the first argument of \cs{edtext} is typset in one line, but it is \textbf{not} reversed when this first argument is typset in two lines or more.\footnote{This problem is caused by the way \XeTeX\ manages right-to-left typsetting. David Carlisle explains it on \url{http://tex.stackexchange.com/a/333373/7712} and provides a potential solution, using \cs{vadjust}. However in some cases this adds spurious vertical spaces in reledmac. That is why we are using the solution explained below.}
 % 
-% To turn over this problem, we use crossref mechanism. At the first run, we put label, but we do not write any \cs{@ref} command. When the value of the label can be tested, that is after three runs, we do this thing:
+% To solve this problem, we use a crossref mechanism. At the first run, we put a label, but we do not write any \cs{@ref} command. When the value of the label can be tested, that is after three runs, we're doing:
 % \begin{itemize}
-%  \item If the first argument of \cs{edtext} is typeset on only one line, we first call \cs{flag@end}, at the point we normally call \cs{flag@start}, at the beginning of the content of the first argument,  and we call \cs{flag@end} at the point we normally call \cs{flag@start}, at the end of the content of the first argument.
+%  \item If the first argument of \cs{edtext} is typeset on only one line, we first call \cs{flag@end}, at the point we normally call \cs{flag@start}, at the beginning of the content of the first argument, and we call \cs{flag@end} at the point we normally call \cs{flag@start}, at the end of the content of the first argument.
 %  \item If the first argument of \cs{edtext} is typeset on only two lines, we use the normal order.
 % \end{itemize}
-% This system compensate the problem of order when writing in auxiliary files.
+% This system fixes the problem of order when writing in auxiliary files.
```

Comment:
I am not sur we can tell it fix the problem. It does not really fix, but avoid it.